### DISCUSSION TITLE
Traffic Bot Use Case

### DISCUSSION THREAD

**Discussion by trafficcreator** [lucasveneno/trafficbot#28] [General]
Hello,
I have seen your traffic bot, and I wanted to know if it is possible to use it for a website and resell traffic with it. (If allowed)
I am looking for a traffic bot like the one www.traffic-bot.com uses. They use datacenter IP, residential ips to generate website traffic and they are also able to generate google search console traffic. I am not really good with coding and I am doing some research how I can sell bot traffic. Would your bot be capable of doing something like that? Which modifications would it need?

And where do ist start/ what type of  IT guys do I need to start something like that?

**Comment by lucasveneno** [comment]
Is totally capable, all you need is a full-stack dev he will be able to work on the front end and the back end of this project without a problem.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: trafficcreator ***@***.***>
Sent: Monday, January 30, 2023 10:08:05 AM
To: lucasveneno/trafficbot ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [lucasveneno/trafficbot] Traffic Bot Use Case (Discussion #28)


Hello,
I have seen your traffic bot, and I wanted to know if it is possible to use it for a website and resell traffic with it. (If allowed)
I am looking for a traffic bot like the one www.traffic-bot.com<http://www.traffic-bot.com> uses. They use datacenter IP, residential ips to generate website traffic and they are also able to generate google search console traffic. I am not really good with coding and I am doing some research how I can sell bot traffic. Would your bot be capable of doing something like that? Which modifications would it need?

And where do ist start/ what type of IT guys do I need to start something like that?

**Reply by trafficcreator** [reply]
That sounds great! Then I will contact someone to do it :)

**Reply by nbitelecom** [reply]
> Is totally capable, all you need is a full-stack dev he will be able to work on the front end and the back end of this project without a problem. Sent from Outlook for Android<https://aka.ms/AAb9ysg>
> […](#)
> ________________________________ From: trafficcreator ***@***.***> Sent: Monday, January 30, 2023 10:08:05 AM To: lucasveneno/trafficbot ***@***.***> Cc: Subscribed ***@***.***> Subject: [lucasveneno/trafficbot] Traffic Bot Use Case (Discussion #28) Hello, I have seen your traffic bot, and I wanted to know if it is possible to use it for a website and resell traffic with it. (If allowed) I am looking for a traffic bot like the one [www.traffic-bot.com<http://www.traffic-bot.com](http://www.traffic-bot.com<http://www.traffic-bot.com)> uses. They use datacenter IP, residential ips to generate website traffic and they are also able to generate google search console traffic. I am not really good with coding and I am doing some research how I can sell bot traffic. Would your bot be capable of doing something like that? Which modifications would it need? And where do ist start/ what type of IT guys do I need to start something like that? — Reply to this email directly, view it on GitHub<#28>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABN6YHEJYRBCCNWQR4JHPILWU7DMLANCNFSM6AAAAAAULGG6G4>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Share me Ur contact info How Can i talk to u willing to work along side or i want some one to help me out Linux ubuto is no problem I want some one to help me Out and have better understanding on the project  Can we lower down the bandwidth Consumption On this Let me know Share me ur skype ID or email or what's app

**Comment by sdesha23** [comment]
Is it safe with google AdSense ?
